samples/samples-react/README.md

```diff
@@ -0,0 +1,41 @@
+# React
+
+This is an example how to use `xlsx-import` in React.
+
+**LIB Version:** `2.3.1`
+
+## Usage
+
+```bash
+# install dependencies
+npm install
+
+# start dev server
+npm start
+
+# 
+```
+
```

Comment:
Please add information about useful scripts, like run cypress test runner, start the app, etc